### PR TITLE
[READY] Adds the Empath and Friendly quirks

### PR DESCRIPTION
**Pull request by Mickyan**
:cl:
add: New quirk: Empath. Gain a better insight on the state of people around you by examining them.
add:  New quirk: Friendly. Your hugs are just the best. Keep yourself in a good mood to give the bestest hugs!
/:cl:

Friendly (1 pt) gives a higher mood bonus whenever you hug someone so long as you have neutral/good mood. The bonus will be even greater if you manage to keep your mood above the top threshold.

Empath (2 pts) gives very vague messages on the state of whoever you're examining. 
You can learn the following from examination:
-Not on help intent
-Oxyloss over 10
-Toxloss over 10
-Negative mood (will give a mild mood malus to the empath)
-Is blind
-Is deaf

I wanted some quirks centered around interacting with others in positive ways. Friendly in particular you're sacrificing a point for no personal gain, so you really have to be friendly to pick it! 
Empath can also be something of a crutch for doctors as it gives immediate feedback on what's wrong with someone even at a distance.

### CODE REVIEW

**Comment by nicbn** [comment]
Empath should make your mood lower if low mood people are around and increase if high mood people are around, it'd make much more sense
The empath you propose should be called behaviourist or something like that

**Comment by OverlyGeneric** [comment]
Can you replace the grab intent messages with the Friendly Quirk from a light grip to gentle embrace, from an aggressive grab to bear hug, and instead of choking them to death you're that huggin' good at hugs that you're turning their face blue? : )

**Comment by MrPerson** [comment]
You begin to hug [victim]'s neck!

**Comment by anconfuzedrock** [comment]
"; THIS GUY IS STARVING AND HE'S STILL IN A GOOD MOOD HE MUST BE AN ANTAG"

**Comment by WJohn** [comment]
Empath seems like an actually useful perk. 👍

**Comment by OverlyGeneric** [comment]
> "; THIS GUY IS STARVING AND HE'S STILL IN A GOOD MOOD HE MUST BE AN ANTAG"

then Gandhi was an antagonist?

**Comment by WJohn** [comment]
Focused is +12 while starving is -15 so you almost definitely won't be in a good mood anyway.

**Comment by anconfuzedrock** [comment]
I thought that was the reasoning for fucking mind reader not letting you actually read minds. maybe I'll add that.

**Comment by PKPenguin321** [comment]
mood probably shouldnt be a thing

**Comment by YakumoChen** [comment]
It is pretty easy to get mood high anyway, even if starving.

File: code/modules/mob/living/carbon/human/examine.dm [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/mob/living/carbon/human/examine(mob/user) //User is the person being examined
+/mob/living/carbon/human/examine(mob/user)
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/mob/living/carbon/human/examine.dm]: why did you remove the comment

Comment (Mickyan) [reply code/modules/mob/living/carbon/human/examine.dm]: oh yeah I forgot to mention it, it's wrong/outdated

user is the examin-er, src the examin-ed

**Comment by ExcessiveUseOfCobblestone** [comment]
> I thought that was the reasoning for fucking mind reader not letting you actually read minds. maybe I'll add that.

letting you see objectives and noticing someone isn't in a poor mood because they're hungry is a bit of a stretch to compare...

The mood boost from just being an antag is a bandaid anyways.

**Comment by ExcessiveUseOfCobblestone** [comment]
also the trait doesn't show mood?

**Comment by Mickyan** [comment]
Yes, that is one of the reasons I used sanity and not mood, there's too many moving parts to sanity to make any kind of reliable assumption from it. 

Besides, you can already figure someone's sanity from whether or not they're running at full speed, which is to say yes the antag mood boost is a bandaid
